docs/root/intro/version_history.rst

```diff
@@ -29,6 +28,7 @@ Version history
   its behaviour within TCP and HTTP implementations.
 * stream: renamed `perRequestState` to `filterState` in `StreamInfo`.
 * thrift_proxy: introduced thrift rate limiter filter
+* tracing: added support for Zipkin tracer of b3 single header format.
```

Comment:
can you ref link to the new header and/or tracing section for more info?

grammer nit: "added support to the Zipkin tracer for the b3 single header format"

Reply:
fixed